message/model.json

```diff
@@ -228,19 +275,48 @@
                             },
                             "type": {
                               "name": "type",
-                              "type": "string"
+                              "type": "string",
+                              "default": "uritemplate",
+                              "required": false
                             },
                             "value": {
                               "name": "value",
-                              "description": "The uri value template",
+                              "description": "The URI value constraint",
                               "type": "uritemplate"
                             },
                             "required": {
                               "name": "required",
-                              "description": "The uri required",
                               "type": "boolean",
-                              "required": true,
-                              "default": false
+                              "default": false,
+                              "required": false
+                            }
+                          }
+                        },
+                        "datacontenttype": {
+                          "name": "CloudEvents datacontenttype https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#datacontenttype.",
+                          "description": "The content type of the data attribute value. ",
+                          "type": "object",
+                          "attributes": {
+                            "description": {
+                              "name": "description",
+                              "type": "string"
+                            },
+                            "type": {
+                              "name": "type",
+                              "type": "string",
+                              "default": "string",
+                              "required": false
```

Comment:
just curious, is there a reason to have: required:false  in some but not others?